COMMITTERS.md

```diff
@@ -0,0 +1,230 @@
+Committing changes to Puppet Agent
+====
+
+This document is liberally copied/based on the same in
+[Puppet](https://github.com/puppetlabs/puppet).
+
+We would like to make it easier for community members to contribute to Puppet
+Agent using pull requests, even if it makes the task of reviewing and
+committing these changes a little harder.  Pull requests are only ever based on
+a single branch, however, we maintain more than one active branch.  As a result
+contributors should target their changes at the master branch. This makes the
+process of contributing a little easier for the contributor since they don't
+need to concern themselves with the question, "What branch do I base my changes
+on?"  This is already called out in the
+[CONTRIBUTING.md](https://goo.gl/XRH2J).
```

Comment:
It looks like this points back to the puppet repo. Maybe we should update or remove this?